### DISCUSSION TITLE
Does it uses `Deno.file`, `Bun.file` under the hood?

### DISCUSSION THREAD

**Discussion by JulianCataldo** [humanwhocodes/humanfs#27] [Q&A] [answered]
Hello,

Is the end-goal of this library to also offer an agnostic layer, by smoothing out runtime specifics?

Thx!

**Comment by nzakas** [comment, marked as answer]
> Does it uses Deno.file, Bun.file under the hood?

No. You can always look at the source code to verify.

> Is the end-goal of this library to also offer an agnostic layer, by smoothing out runtime specifics?

Yes. The idea is that anywhere you want to run JavaScript, there will be an fsx runtime package that will work for you.
